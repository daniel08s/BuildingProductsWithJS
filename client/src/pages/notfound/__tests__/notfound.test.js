/* global test expect shallow React */
import Notfound from '../index';

test('# Notfound', () => {
  const wrapper = shallow(<Notfound />);
  expect(wrapper).toMatchSnapshot();
});
