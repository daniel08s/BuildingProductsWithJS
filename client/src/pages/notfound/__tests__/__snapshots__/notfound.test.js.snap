// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# Notfound 1`] = `
<h1>
  Page not found
</h1>
`;
